src/csharp/Grpc.Core/Grpc.Core.csproj

```diff
@@ -67,6 +67,12 @@
     <PackageReference Include="System.Threading.ThreadPool" Version="4.0.10" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="System.Runtime.Loader" Version="4.0.0" />
+    <PackageReference Include="System.Threading.Thread" Version="4.0.0" />
+    <PackageReference Include="System.Threading.ThreadPool" Version="4.0.10" />
```

Comment:
I think this PR should be re-opened so that projects compatible with netstandard2.0 don't end up with these transitive package references.

Reply:
what should they end up instead?